tests/smoke/base.sh

```diff
@@ -70,8 +70,16 @@ $COMMAND_PREFIX pre-commit install
 
 $COMMAND_PREFIX pre-commit run --all-files --verbose
 
-$COMMAND_PREFIX make docs-build
-make build
+if "{{cookiecutter.automation_tools}}" == "makim" ; then
```

Comment:
sorry just remembered that the cookiecutter variables are not accessable here
so you need to use something like:

```suggestion
if command -v makim &> /dev/null; then
```